python/skupper_router/management/skrouter.json

```diff
@@ -368,6 +368,143 @@
             "attributes": {}
         },
 
+        "routerStats": {
```

Comment:
Instead of "routerStats" what do you think about "routerMetrics"?  I'd run this past The Grand Poobah of All Things Naming  (@ssorj).  Just an idea - not a request.

Reply:
Renamed routerStats to routerMetrics - done